configs/AM335X/AM335X_linux_toc.txt

```diff
@@ -143,6 +143,7 @@ linux/How_to_Guides/Target/Processor_SDK_Linux_File_System_Optimization_Customiz
 linux/How_to_Guides/Target/How_to_add_a_JVM
 linux/How_to_Guides/Target/How_to_use_a_Mouse_instead_of_the_Touchscreen_with_Matrix
 linux/How_to_Guides/Target/Update_U-Boot_Environment_Variables_stored_in_SPI_Flash_from_Linux
+linux/How_to_Guides/Target/How_to_fix_y2k38
 linux/How_to_Guides/Target/How_to_Change_dtb_File
```

Comment:
This is a really great start, thank you for putting this together!

I would like to see this page for AM57x as well. I know the team claimed that the out-of-the-box filesystem in SDK 9.2 was y2038 friendly, not sure if they have done any validation that the same applies to SDK 9.3.

Reply:
Will let @aniket-l comment on the validation status of Y2038 in SDK 9.3 for AM57x